modules/localgov_microsites_news/config/optional/field.field.node.localgov_news_article.lgms_banner.yml

```diff
@@ -21,6 +22,7 @@ settings:
   handler: 'default:paragraph'
   handler_settings:
     target_bundles:
+      localgov_banner_primary: localgov_banner_primary
       localgov_banner_secondary: localgov_banner_secondary
     negate: 0
     target_bundles_drag_drop:
```

Comment:
I suggest removing all the `target_bundles_drag_drop:` items with `enabled: false`, shortening this to
```yaml
    target_bundles_drag_drop:
      localgov_banner_primary:
        weight: 0
        enabled: true
      localgov_banner_secondary:
        weight: 1
        enabled: true
```